my-bq-app/src/pages/NotFoundPage.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Loving that you added a 'not found' page! Where are you calling this? I've never used it so I'm curious :o

Reply:
I am calling it in another route inside App.js file like this: <Route path='*' element={<Not Found Page />}/> after importing it